internal/cli/cmd/install.go

```diff
@@ -48,39 +48,53 @@ cilium install --context kind-cluster1 --cluster-id 1 --cluster-name cluster1
 	}
 
 	cmd.Flags().StringVarP(&params.Namespace, "namespace", "n", "kube-system", "Namespace to install Cilium into")
+	// Deprecated
```

Comment:
we can mark this flag as deprecated i.e. this flag will still work, but will not show up in help output.

```
	cmd.Flags().MarkDeprecated("cluster-name", "please use abc instead")
```